### PR TITLE
EOS-19427: Create seperate test rpm and add io tests in it (on tmain)

### PR DESCRIPTION
**Pull request by atulsdeshmukh2312**
Signed-off-by: Atul Deshmukh <atul.deshmukh@seagate.com>

### CODE REVIEW

File: cortx-motr.spec.in [hunk 1]
```diff
@@ -39,15 +40,19 @@
...
-%endif
+%endif #with ut
+%endif #with ivt
 
-Name:           @PACKAGE@
-Version:        @PACKAGE_VERSION@
-Release:        %{build_num}_@GIT_REV_ID_RPM@_%{kernel_ver}%{?dist}
+Name:           cortx-motr
+Version:        2.0.0
+Release:        %{build_num}_git1725ba7_%{kernel_ver}%{?dist}
```
Comment (huanghua78) [inline cortx-motr.spec.in:55]: Please revert these changes.
They are replaced in the automake/autoconf step.

File: cortx-motr.spec.in [hunk 2]
```diff
@@ -39,15 +40,19 @@
 %define  configure_cassandra_opts   --with-cassandra
 %endif
 
+%if %{with ivt}
+%define  configure_opts  %{configure_ut_opts} %{?configure_cassandra_opts}
```
Comment (huanghua78) [inline cortx-motr.spec.in:44]: copy & paste error?
Why is the same with the next line?

File: cortx-motr.spec.in [hunk 3]
```diff
@@ -170,15 +175,45 @@ Conflicts: %{name}
...
 %prep
 %setup -q
 
 %build
+%if !%{with ivt}
 %configure %{with_linux} %{with_lustre} %{configure_opts}
 make %{?_smp_mflags}
+%else
+%configure %{with_linux} %{with_lustre} %{configure_opts}
+%endif
```
Comment (huanghua78) [inline cortx-motr.spec.in:197]:
```
%configure %{with_linux} %{with_lustre} %{configure_opts}

%if !%{with ivt}
make %{?_smp_mflags}
%endif
```

If `%configure %{with_linux} %{with_lustre} %{configure_opts}` is common, then it can be moved out of the condition.

File: cortx-motr.spec.in (hunk 1 above)
Comment (atulsdeshmukh2312) [reply cortx-motr.spec.in]: Done 3ce805d

File: cortx-motr.spec.in (hunk 3 above)
Comment (atulsdeshmukh2312) [reply cortx-motr.spec.in]: 3ce805d

**Comment by huanghua78** [comment]
I  think this is done in #706.
